app/src/main/java/com/goayo/debtify/model/Group.java

```diff
@@ -135,4 +141,25 @@ public String getGroupName() {
     public List<IDebtData> getDebts() {
         return new ArrayList<>(groupLedger.getDebtDataList());
     }
+
+    /**
+     * Fetches and calculates the users with the given phonenumbers remaining total debt.
```

Comment:
should be singular since it's only one user with one phone number,
use "phonenumber's" if it's the phone number belonging to the user.